pages/trust-admin.js

```diff
@@ -163,24 +175,41 @@ export const getServerSideProps = propsWithContainer(
     const retrieveWardVisitTotals = await container.getRetrieveWardVisitTotals()(
       authenticationToken.trustId
     );
-    const averageParticipantsInVisitResponse = await container.getRetrieveAverageParticipantsInVisit()(
+    const {
+      averageParticipantsInVisit,
+      error: averageParticipantsInVisitError,
+    } = await container.getRetrieveAverageParticipantsInVisit()(
+      authenticationToken.trustId
+    );
+
+    const {
+      averageVisitTimeSeconds,
+      error: averageVisitTimeSecondsError,
+    } = await container.getRetrieveAverageVisitTimeByTrustId()(
       authenticationToken.trustId
     );
 
+    const error =
+      wardError ||
+      trustError ||
+      averageParticipantsInVisitError ||
+      averageVisitTimeSecondsError;
+
+    const averageVisitTime = moment
+      .duration(averageVisitTimeSeconds, "seconds")
+      .format("h[hr], m[min]");
```

Comment:
What do we think about adding a space between the number and `hr`/`min`?

![image](https://user-images.githubusercontent.com/42817036/86016610-ead58800-ba1a-11ea-849e-cdd661b81db7.png)

Reply:
Good shout that reads much better